src/vos/tests/vts_ts.c

```diff
@@ -761,7 +761,10 @@ static int
 ts_test_fini(void **state)
 {
 	struct ts_test_arg	*ts_arg = *state;
+	struct vos_ts_table	*ts_table;
 
+	ts_table = vos_ts_table_get();
+	vos_ts_table_free(&ts_table);
```

Comment:
Is this a current memory leak?  Is so I'd like to try this with my leak checker before we fix it.